app/models/ChangeIndicators.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json._
+
+case class ChangeIndicators(organisationDetails: Boolean,
+                            ppob: Boolean,
```

Comment:
Some of these field names are incorrect with respect to what they are in the JSON which is causing this model to always be read as `None`. For example, I tried user 247246260 who has `organisationDetails` pending but it let me into the service.

This service doesn't care about any other pending sections, just org details, so an easy fix to the issue would be to get rid of the other booleans here and just keep `organisationDetails`. Try hitting one of the URLs directly as user 247246260 as a way to manually test this.